terraform/modules/function/main.tf

```diff
@@ -10,6 +10,14 @@ locals {
   }
 }
 
+data "aws_kms_alias" "bcda_app_config_kms_key" {
+  name = "alias/bcda-${var.env}-app-config-kms"
+}
```

Comment:
Instead of making a bcda-specific call here, the module should allow for additional key IDs to be passed in. Then in the services that include BCDA make the data call and pass in the ID.